README.md

```diff
@@ -102,6 +102,8 @@ If you get `nvm: command not found` after running the install script, one of the
 
   - You might need to restart your terminal instance or run `. ~/.nvm/nvm.sh`. Restarting your terminal/opening a new tab/window, or running the source command will load the command and the new configuration.
 
+  - Previously used `bash`, but with `zsh` installed. You need to write the configuration to `~/.zshrc` and run `source ~/.zshrc`.
```

Comment:
Can you elaborate on this scenario?

Reply:
macOS default command line tools is `bash`, but I have installed `oh my zsh`. When i run the installation command, open the new command line tools and type `nvm -l`,  it will prompt `nvm: command not found`.

When I write the configuration to `.zshrc `,  type 'nvm-l' and it will work.

Reply:
Following the steps in https://github.com/nvm-sh/nvm#troubleshooting-on-macos would already lead to "Try adding the snippet from the install section", which should have fixed things for you. Did it not?

Reply:
Sorry, I didn't notice, but why not put it together with the above.